### PR TITLE
[13.0][MIG] l10n_es

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Migration of l10n_es.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/openupgrade_records/lib/apriori.py [hunk 1]
```diff
@@ -37,6 +37,7 @@
     # OCA/account-reconcile
     'account_set_reconcilable': 'account',
     # OCA/l10n-spain
+    'l10n_es_account_invoice_sequence': 'account',
```
Comment (MiquelRForgeFlow) [inline odoo/addons/openupgrade_records/lib/apriori.py]: delete this, as is already merged in l10n_es (repeated key)

File: addons/l10n_es/migrations/13.0.4.0/pre-migration.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def set_account_move_number_to_invoice_number(env):
+    """l10n_es_account_invoice_sequence introduced different numbering between
+    invoices and account moves.
+
```
Comment (MiquelRForgeFlow) [inline addons/l10n_es/migrations/13.0.4.0/pre-migration.py]:
```suggestion

```

File: odoo/addons/openupgrade_records/lib/apriori.py (hunk 1 above)
Comment (pedrobaeza) [reply odoo/addons/openupgrade_records/lib/apriori.py]: I'm seeing, thanks.

**Review by MiquelRForgeFlow — COMMENTED** [review]
:+1:

**Pull request merged**